modal/sandbox.py

```diff
@@ -73,6 +73,12 @@ def _validate_exec_args(args: Sequence[str]) -> None:
         )
 
 
+def _validate_sandbox_name(name: str) -> None:
+    message = f"Invalid Sandbox name: '{name}'.\n\nSandbox names must be between 1 and 128 characters long."
+    if len(name) == 0 or len(name) > 128:
```

Comment:
Note: we're not using the `name_utils` validation functions because Sandbox names can (and often are) longer than 64 characters. This puts in place the same constraints we had at Jamsocket and that are currently set on the DB schema.